python/ray/tune/trial.py

```diff
@@ -10,8 +10,31 @@
 from ray.rllib.agent import get_agent_class
 
 
-# Ray resources required to schedule a Trial
-Resources = namedtuple("Resources", ["cpu", "gpu"])
+class Resources(
+        namedtuple("Resources", ["cpu", "gpu", "driver_cpu", "driver_gpu"])):
```

Comment:
why do we want to keep this as a named tuple?

Reply:
Hm that gives you some nice properties for free, basically it's like a struct as a base class.